frontend/src/components/message.module.css

```diff
@@ -25,11 +23,50 @@
   width: 40px;
   height: 40px;
   margin: 16px;
-  align-self: flex-start;
 }
 
 .messageStyle {
   margin: 24px 32px 16px 0;
   line-height: 22px;
   font-size: 16px;
 }
+
+.reasoning_header {
+  font-weight: bold;
+  font-size: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-top: 1px solid var(--grey-300);
+  padding: 0 24px 0 16px;
+  height: 40px;
+}
+
+.reasoning_header:hover {
```

Comment:
Do we need some css on focus as well here?

![image](https://github.com/user-attachments/assets/3e12fe8f-9c00-41ee-80cc-0247818f97a1)

Reply:
> Do we need some css on focus as well here?
> 
> ![image](https://private-user-images.githubusercontent.com/9381751/389491896-3e12fe8f-9c00-41ee-80cc-0247818f97a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NDQzOTksIm5iZiI6MTczMjU0NDA5OSwicGF0aCI6Ii85MzgxNzUxLzM4OTQ5MTg5Ni0zZTEyZmU4Zi05YzAwLTQxZWUtODBjYy0wMjQ3ODE4Zjk3YTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjVUMTQxNDU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTg5YTE1MzgwMjQ1ZGQ4OGVjMjdjMWViYjcxZmYyZWI4NzRjMDQ5ZGRhOTY1ZjE5Zjc3YjBhMjkxNGEzZWZjMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.76acbL_u76pPLsCtMC5Xohci7uMaSdz8pP0pSA85lEQ)

This is consistent with the other keyboard focus handling. But I'll check and raise a follow up ticket if needed so not blocking this